### PR TITLE
add awadb vectorstore

### PR DESCRIPTION
**Pull request by ljeagle**
@dev2049 Add a new vectorstore-AwaDB

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -103,6 +103,7 @@ py-trello = {version = "^0.19.0", optional = true}
 momento = {version = "^1.5.0", optional = true}
 bibtexparser = {version = "^1.4.0", optional = true}
 pyspark = {version = "^3.4.0", optional = true}
+awadb = {version = "^0.3.1", optional = true}
```
Comment (ljeagle) [inline pyproject.toml]: update the latest version 0.3.2

File: poetry.lock [hunk 2]
```diff
@@ -566,6 +570,30 @@ dev = ["coverage (>=5,<6)", "flake8 (>=3,<4)", "pytest (>=6,<7)", "sphinx-copybu
...
 
+[[package]]
+name = "awadb"
+version = "0.3.1"
+description = "AI Native database for embedding vectors"
+category = "main"
+optional = true
+python-versions = ">=3.7"
+files = [
+    {file = "awadb-0.3.1-cp310-cp310-macosx_10_16_universal2.whl", hash = "sha256:bebc2125665f19145761667880127ff891ef160be59a15d8b187d7ca54bd2b30"},
```
Comment (ljeagle) [inline poetry.lock:581]: Update the latest version of awadb to 0.3.2

**Review by ljeagle — COMMENTED** [review]
Use the latest version of awadb from 0.3.1 to 0.3.2

**Review by hwchase17 — COMMENTED** [review]
we should add a notebook in the integrations section highlighting this

**Pull request closed without merging**